charts/newrelic-logging/Chart.yaml

```diff
@@ -18,5 +18,5 @@ keywords:
   - newrelic
 dependencies:
   - name: common-library
-    version: 0.14.3
+    version: 1.0.4
     repository: file://../../library/common-library
```

Comment:
I did not realised previously that you were using a local path...

This has to be a URL like in other charts: https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/Chart.yaml#L14-L17

This is because, in case you need to migrate the source code somewhere else, you can still publish the chart here during the EOL time.

It removes also issues with possible differences between the local filesystem and the CI filesystem.

Reply:
No worries, I'll change this. When I opened the PR, common had not been published / used throughout the code yet, so I went with the example in the library folder. That line is certainly more recent than this commit.

I don't think the FS matters too much as it's a relative path, but I'm not arguing :-) 

I'll tidy up the last few bits when it's not so late in the evening! Neat to see the common ;ibrary chart being used, I'll certainly be taking a bit of inspiration from your setup in my own work, thanks for your time reviewing this @kang-makes 💯

Reply:
If you even want to use that common library for your own work feel free to use ours and contribute to it. Open an issue before jumping to develop a feature in case we do an effort that is not going to be accepted so we can analyze it and we will be happy to merge it :)

Here you have an example: https://github.com/newrelic/helm-charts/pull/794#issuecomment-1118795491

You have other examples of charts using libraries in the Bitnami charts or the k8s-at-home project if you wish to take a look at them.

I am eager for you to solve these issues and merge it, we had an approval from the logging team :)

Reply:
Changed URL and updated the chart lock